blueflood-core/src/main/java/com/rackspacecloud/blueflood/inputs/formats/AggregatedPayload.java

```diff
@@ -61,18 +55,15 @@ public String toString() {
     public Collection<BluefloodSet> getSets() { return safeAsList(sets); }
     public Collection<BluefloodEnum> getEnums() { return safeAsList(enums); }
 
-
-    public List<String> getValidationErrors() {
-
-        List<String> errors = new java.util.ArrayList<String>();
-        long current = System.currentTimeMillis();
-
-        if( timestamp > current + futureDiff )
-            errors.add( "'timestamp' '" + timestamp + "' is more than '" + futureDiff + "' milliseconds into the future." );
-        else if( timestamp < current - pastDiff )
-            errors.add( "'timestamp' '" + timestamp + "' is more than '" + pastDiff + "' milliseconds into the past." );
-
-        return errors;
+    public String validate() {
```

Comment:
I don't like splitting it to 2 methods, which was what I had before, because we have the code do that timestamp compare and calculation against the diffs twice for every metric.  I'll rename it.